where-to-travel/src/main/webapp/signup.js

```diff
@@ -0,0 +1,61 @@
+// Get elements for authentication
+const emailSignUp = document.getElementById('emailSignUp');
+const passwordSignUp = document.getElementById('passwordSignUp');
```

Comment:
Since most of these global constants are only used in 1 function, could we just define them in that function scope? Usually, avoid globals as much as possible.

Reply:
done